### ISSUE TITLE
Crawl data from Cooper Hewitt Museum

### ISSUE THREAD

**Issue by ehrhart** [silknow/crawler#18]
https://collection.cooperhewitt.org/departments/35347501/ should show their textile department’s collection, but strangely it shows nothing, when it’s clear that they do have a excellent collection. 
Their catalog search is also quite weak and unpredictable.

They do have an API, on the other hand, but it’s hard to provide searching strategies without any hint on the expected results.
https://collection.cooperhewitt.org/api/

Probably it’s not worth the effort, which is sad, since the collection is very good.

UPDATE: May 14, 2020
I met one of their digital analysts in NYC last February. She told me that, contrary to my assumptions, they do have a good historical collection, not just 20th century. However, their online interface is still very impractical, making searches a hit-or-miss affair. 

Could their API be more practical for us, allowing queries incorporating geographic (Europe) and chronological (14th to 19th centuries) limitations? On the standard search engine, this seems impossible.

On the other hand, while they do offer all their data as open access, they explicitly exclude images. However, they do endorse fair use of all their materials. I would be surprised if we got into trouble with them, and in any case not for pre-20th century holdings, where any copyright restrictions don’t apply, as a rule.
https://www.cooperhewitt.org/open-source-at-cooper-hewitt/

Summing up: better use the resources from the Smithsonian (see below).

UPDATE: June 21, 2020
They have 2 very interesting sample books from Italy, mid-19th century. Each of them is fully photographed, summing up some 100 images. How can we incorporate those records and all the associated images?
https://collection.cooperhewitt.org/objects/18766645/
https://collection.cooperhewitt.org/objects/18766647/

---

Addendum for records about paper-based items (point paper plans, drawings, etc.)

Same search interface problems as reported above.
They do have some useful records: see, for example, https://collection.cooperhewitt.org/objects/18102947/

See also [gdoc reference](https://docs.google.com/document/d/1Tg4gDlz8HjILPRQ8XBOBFo-3-74nmJdJ31YL6PUtdB4/edit#heading=h.345xxyyu0xaj)

**Comment by rtroncy** [comment]
On the [30/04/2021 meeting](https://docs.google.com/spreadsheets/d/1XutXqbPQC5_hUeg_B7gzUh5A2C_zCmKnWLLjNCCG7WU/edit#gid=937636458), we decided to not consider this museum as the data provided is of less good quality of what we already have as part of the Smithsonian we have already harvested
